openforcefield/topology/molecule.py

```diff
@@ -2861,6 +2884,7 @@ def from_file(filename,
 
         # Determine which toolkit to use (highest priority that's compatible with input type)
         if isinstance(toolkit_registry, ToolkitRegistry):
+            # TODO: Encapsulate this logic into ToolkitRegistry.call()?
```

Comment:
Ah, this could be a really good idea. We'd need to be careful, however, about the case where RDKit skips a molecule (because then we'd want to try to use another toolkit for it)